styles/styles.css

```diff
@@ -393,10 +396,26 @@ main .block.lead p {
   width: 30%!important;
 }
 
+.float-left {
+  float: left;
+}
+
 .float-right {
   float: right;
 }
 
+.padded-block {
+  padding: 1rem 0!important;
+}
+
+.section.top-65 > .section-container {
+  top: 65% !important;
```

Comment:
`!important` should usually be avoided unless absolutely needed. Can we manage without them?

Reply:
Yes,` !important `is needed to override the more specific selectors. 

<img width="831" alt="Screenshot 2023-12-05 at 2 40 35 PM" src="https://github.com/hlxsites/sunstar/assets/119420493/b7ae3ed6-066f-4293-9b3c-fc6da53f123a">

Reply:
ok, would using a more specific selector help?

Reply:
I have to go with the following selector if we have to remove `!important`, and that may not look generic to be used across.

`.section.with-background-image.align-center.top-65 > .section-container {
  top: 65%;
}`

I can add with this if we are fine having this selector.

Reply:
or maybe remove `align-center` (that seems to be causing this style)  and have align-bottom or something. But I think removing `!important` is good idea in any case.

Reply:
Yes, `align-bottom` requires a little adjustment for this page.  I have changed` top:80%` to `top:75%` for this page.
With `top:80%`, text was spill over (snapshot attached). 
<img width="1003" alt="Screenshot 2023-12-06 at 12 45 17 AM" src="https://github.com/hlxsites/sunstar/assets/119420493/8473c924-34f0-44a2-9af4-f371104795a5">

After changing to top:75%, it looks good for the https://main--sunstar--hlxsites.hlx.live/healthy-thinking/penguin-indoor-seal page as well. CC: @dnbute